frontend/src/old-pages/Clusters/Clusters.tsx

```diff
@@ -112,53 +119,43 @@ function ClusterList() {
         </Header>} trackBy="clusterName" columnDefinitions={[
         {
             id: "name",
-            header: "Name",
+            header: t("cluster.list.cols.name"),
             cell: item => (item as any).clusterName,
             sortingField: "clusterName"
         },
         {
             id: "status",
-            header: "Status",
+            header: t("cluster.list.cols.status"),
             cell: item => <Status status={(item as any).clusterStatus} cluster={item}/> || "-",
             sortingField: "clusterStatus"
         },
         {
             id: "version",
-            header: "Version",
+            header: t("cluster.list.cols.version"),
             cell: item => (item as any).version || "-"
         }
     
     ]} 
     loading={clusters === null}
     items={items}
     selectionType="single"
-    loadingText="Loading clusters..."
+    loadingText={t("cluster.list.loadingText")}
     pagination={<Pagination {...paginationProps}/>}
     filter={<TextFilter {...filterProps}
-    countText={`Results: ${filteredItemsCount}`}
-    filteringAriaLabel="Filter cluster"/>}
+    countText={`${t("cluster.list.countText")} ${filteredItemsCount}`}
+    filteringAriaLabel={t("cluster.list.filteringAriaLabel")}/>}
     selectedItems={(items || []).filter((c) => (c as any).clusterName === selectedClusterName)}
-    // @ts-expect-error TS(2571) FIXME: Object is of type 'unknown'.
-    onSelectionChange={({ detail }) => { navigate(`/clusters/${detail.selectedItems[0].clusterName}`); }}
+    onSelectionChange={({ detail }) => { navigate(`/clusters/${(detail.selectedItems[0] as Cluster).clusterName}`); }}
```

Comment:
nitpick: It may be a good idea to extract this and put it into a [useCallback](https://reactjs.org/docs/hooks-reference.html#usecallback)